### PR TITLE
Feature: Claim rewards

### PR DESCRIPTION
**Pull request by Vheissu**
Allow users to claim pending rewards for Steem Engine tokens. Adds in the ability to claim tokens above balances as well as individual tokens themselves (if they're in the list). Uses custom JSON operations to perform the claiming functionality.

### CODE REVIEW

File: scripts/Config-Dev.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ Config = {
 	RPC_URL: 'https://testapi.steem-engine.com/',
 	ACCOUNTS_API_URL: 'https://testaccounts.steem-engine.com',
 	CONVERTER_API: 'https://converter-api.steem-engine.com/api',
+	SCOT_API: 'http://scot-api.steem-engine.com/',
```
Comment (holgern) [inline scripts/Config-Dev.js]: The https address works also:
SCOT_API: 'https://scot-api.steem-engine.com/',

File: scripts/SE.js [hunk 2]
```diff
@@ -428,10 +429,55 @@ SE = {
...
+		const amount = SE.User.ScotTokens[symbol].pending_token;
+		const factor = Math.pow(10, token.precision);
+		const calculated = amount / factor;
+
+		const claimData = {
+			symbol
+		};
+
+    if (useKeychain()) {
+      steem_keychain.requestCustomJson(username, 'scot_claim_token', 'Active', JSON.stringify(claimData), `Claim ${calculated} ${symbol.toUpperCase()} Tokens`, function(response) {
```
Comment (holgern) [inline scripts/SE.js]: the custom_json can also be sent with the posting authority

File: scripts/SE.js [hunk 3]
```diff
@@ -428,10 +429,55 @@ SE = {
...
+		SE.ShowLoading();
+		
+		const token = SE.Tokens.find(t => t.symbol === symbol);
+		const username = SE.User.name;
+		const amount = SE.User.ScotTokens[symbol].pending_token;
+		const factor = Math.pow(10, token.precision);
+		const calculated = amount / factor;
+
+		const claimData = {
+			symbol
```
Comment (holgern) [inline scripts/SE.js]: claimData should be
{"symbol": symbol}

Comment (Vheissu) [reply scripts/SE.js]: The way I've done it works, this is an ES6 feature which restructures the value, it makes it so you don't have to redundantly define nothing the key and property if they're the same name. I do this for the Scotbot initialisation and settings app as well.

File: scripts/Config-Dev.js (hunk 1 above)
Comment (Vheissu) [reply scripts/Config-Dev.js]: Good catch. This should be https

File: scripts/SE.js (hunk 2 above)
Comment (Vheissu) [reply scripts/SE.js]: Is there any disadvantage using one or the other?

Comment (holgern) [reply scripts/SE.js]: Using the posting authority is more secure, as the active key is not used, thats all.

**Pull request merged**